<!doctype html>
<html lang="en" itemscope itemtype="http://schema.org/Person">
<head>
  <meta charset="utf-8">
  <!-- Site Meta Data -->
  <title>Research</title>
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="description" content="">
  <meta name="author" content="Fabio Zanini">

  <link rel="shortcut icon" href="/images/favicon.ico">

  <!-- schema.org -->
  <meta itemprop="name" content="fabilab">
  <meta itemprop="image" content="https://fabilab.org/images/logo2.png">
  <meta itemprop="description" content="">

  <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,600,700' rel='stylesheet' type='text/css'>
  <!-- Style Meta Data -->
  <link rel="stylesheet" href="https://fabilab.org/theme/css/style.css" type="text/css" />
  <link rel="stylesheet" href="https://fabilab.org/theme/css/pygments.css" type="text/css" />

  <!-- Feed Meta Data -->
    <link href="https://fabilab.org/feeds/all.atom.xml" type="application/atom+xml" rel="alternate" title="fabilab ATOM Feed" />

  <!-- Twitter Feed -->
  <meta name="twitter:card" content="summary">
  <meta name="twitter:site" content="">
  <meta name="twitter:image" content="">
</head>

<body>
  <!-- Sidebar -->
  <aside>
    <center><a href="https://fabilab.org"><img id="avatar" src="https://fabilab.org/images/logo2.png"></a></center>
    <h1><a id="sidetitle" href=https://fabilab.org>fabilab</a></h1>
    <div class="logosub">Data driven biomedicine<br>UNSW SYDNEY, AUSTRALIA</div>
    <br>


    <div id="menuArrowDown"><img src="https://fabilab.org/theme/images/icons/arrowdown.svg" /></div>
    <nav class="nav">
      <ul class="list-bare">
         
          <li><a class="nav__link" href="https://fabilab.org/">RESEARCH</a></li>
          <li><a class="nav__link" href="https://fabilab.org/pages/people.html">PEOPLE</a></li>
          <li><a class="nav__link" href="https://fabilab.org/pages/teaching.html">TEACHING</a></li>
          <li><a class="nav__link" href="https://fabilab.org/pages/software.html">SOFTWARE</a></li>
          <li><a class="nav__link" href="https://fabilab.org/pages/papers.html">PAPERS</a></li>
          <li><a class="nav__link" href="https://fabilab.org/pages/contact.html">CONTACT</a></li>
          <li><a class="nav__link" href="https://fabilab.org/pages/jointhelab.html">JOIN</a></li>
        <li><a class="nav__link" href="https://fabilab.org/news.html">NEWS</a></li>
         
      </ul>
    </nav>

    <p class="social">
        <a href="https://twitter.com/fabiousername" target="_blank" ><img src="https://fabilab.org/theme/images/icons/twitter.png"></a>
        <a href="https://github.com/iosonofabio" target="_blank" ><img src="https://fabilab.org/theme/images/icons/github.png"></a>
      <a href="https://fabilab.org/feeds/all.atom.xml" rel="alternate">
        <img src="https://fabilab.org/theme/images/icons/rss.png"></a>
    </p>

    <!--
    <h2>Categories</h2>
    <ul class="navbar">
      <li><a href="https://fabilab.org/category/blog.html">blog</a></li>
    </ul> 
    -->
  </aside>

  <!-- Content -->
  <article>
<section id="content" class="body">
        

        <!-- first block is the PI, keep that -->
        
        <!-- other folks, put several per row, depending on media width -->

        <h3>We give a voice to each single cell of the body to show that diversity is an asset, not a&nbsp;liability.</h3>
<h4><span class="caps">OPEN</span> <span class="caps">POSITION</span> for a paid intern on network analysis charts in igraph and Python. Nov 2021-Jan 2022 <a href="https://fabilab.org/images/intern_fabilab_igraph_2021.pdf">Project&nbsp;description</a></h4>
<h4><span class="caps">OPEN</span> <span class="caps">POSITION</span> for a master/honours/PhD stuent shared with <a href="http://www.oceanography.unsw.edu.au/">Amandine Schaeffer</a> on gene stealing in bluebottle jellyfish. <a href="https://fabilab.org/images/bluebottle_single_cell_project.pdf">Project&nbsp;description</a></h4>
<h1>Research</h1>
<p><a href="pages/contact.html">Fabilab</a> is driven by <a href="pages/people.html">people</a>, not <a href="https://en.wikipedia.org/wiki/Interdisciplinarity">academic boundaries</a>. We proudly work on biology, medicine, data science, computer science, network science and, sometimes,&nbsp;math.</p>
<h2>Gene networks regulating leukemic cell state&nbsp;transitions</h2>
<p><img alt="Embedding AML cells" src="https://fabilab.org/images/research/umap_aml.png"></p>
<p><strong>Cancer cells are not equal in the face of therapeutics.</strong> Within a single patient, even within a single Petri dish, heterogeneity in gene expression and function distinguishes more stem-like cells, which are more likely to cause relapse, from more differentiated cells. Using our <a href="https://www.nature.com/articles/s41598-020-71805-1">northstar</a> algoritm, we developed <a href="https://ashpublications.org/blood/article-abstract/doi/10.1182/blood.2020009707/476046/Disruption-of-a-GATA2-TAL1-ERG-regulatory-circuit?redirectedFrom=fulltext">data exploration techniques</a> to reveal how the transition between these cancer cell states is regulated and how it can be perturbed chemically, with the vision to reduce relapse in acute myeloid leukemia (<span class="caps">AML</span>).</p>
<h2>Biology of the neonatal&nbsp;lung</h2>
<p><img alt="Neonatal lung immunity" src="https://fabilab.org/images/research/lung_immune.png"></p>
<p><strong>The lung is a very special organ at birth</strong>, carrying pathogens and chemically reactive oxygen right into the center of our bodies. Our lab is <a href="https://elifesciences.org/articles/56890">mapping</a> the <a href="https://www.biorxiv.org/content/10.1101/2021.04.27.441649v1">staggering</a> cellular <a href="https://www.biorxiv.org/content/10.1101/2021.05.19.444776v1.full">diversity</a> characterizing neonatal lungs in terms of gene expression (left) and anatomical location (right). A deeper understanding of this fundamental biological system will help us treat the lung conditions affecting thousands of newborn babies every year, such as bronchopulmonary&nbsp;dysplasia.</p>
<h2>Immunology of severe&nbsp;dengue</h2>
<p><img alt="Severe dengue" src="https://fabilab.org/images/research/severe_dengue.png"></p>
<p><strong>Dengue</strong> is the most widespread mosquito-transmitted viral disease with 400 million infections every year. We are developing <a href="https://elifesciences.org/articles/32942">new</a> <a href="https://www.pnas.org/content/115/52/E12363">approaches</a> to understand what different parts of the immune system are doing during severe dengue. We identified <span class="caps">CD163</span> as a biomarker for severity in monocytes and patented <a href="https://elifesciences.org/articles/52384">new antibodies</a> against dengue&nbsp;virus.</p>
<h2>Graph clustering of tumor&nbsp;transcriptomes</h2>
<p><img alt="northstar" src="https://fabilab.org/images/research/northstar.png"></p>
<p><strong>Cell atlases</strong> are huge collections of single cell transcriptomes that describe in extreme molecular detail the composition of human tissues. We are pioneering <a href="https://www.nature.com/articles/s41598-020-71805-1">new algorithms</a> to leverage cell atlases to rapidly elucidate the composition of tumor&nbsp;biopsies.</p>
<h2>Network&nbsp;science</h2>
<p><img alt="igraph" src="https://fabilab.org/images/research/igraph_with_plot.png"></p>
<p><strong>Graphs or networks are an essential mathematical object in modern systems biology.</strong> Together with some <a href="https://www.traag.net/">amazing</a> <a href="http://szhorvat.net/pelican/">folks</a>, we develop <a href="https://igraph.org/">igraph</a>, a high-performance network analysis <a href="https://www.researchgate.net/profile/Jesus_Cortes-Rodicio/post/Can_anyone_suggest_any_papers_about_bibliometric_software/attachment/59d61ddb79197b807797b0d5/AS:273730993885186@1442273962343/download/iGraph.pdf">tool</a> that underpins many software packages in single cell biology and&nbsp;beyond.</p>
<h2>HTSeq&nbsp;2.0</h2>
<p><img alt="HTSeq" src="https://fabilab.org/images/research/htseq2.png"></p>
<p><strong>HTSeq is a software library to analyse high-throughput sequencing data in Python</strong>. It is particularly popular to quantify gene expression in bulk and single-cell <span class="caps">RNA</span>-Seq data via its <code>htseq-count</code> script. We have been maintaining <em>HTSeq</em> for many years and have developed <a href="https://arxiv.org/abs/2112.00939">HTSeq 2.0</a>, adding specific support for single cell experiments, exon-level expression, and a dedicated new <span class="caps">API</span> to manage islands of dense genomic data sprinkled in data&nbsp;deserts.</p>
</section>
  </article>

  <!-- Footer -->
  <footer>
    <p>
    <img id="aboriginalflag" src="https://fabilab.org/images/australian_aboriginal_flag.jpg"> We acknowledge the <a href="https://en.wikipedia.org/wiki/Bidjigal">Bedegal</a> people that are the Traditional Custodians of the land where the lab stands. We pay our respects to the Elders past and present and extend that respect to other Aboriginal and Torres Strait Islanders who are reading this page. 
    </p>
    <p>
      Site powered by <a href="http://getpelican.com/">Pelican</a>, 
      Theme <a href="https://github.com/parbhat/pelican-blue">Pelican-Blue</a> by <a href="https://parbhatpuri.com/">@parbhat</a>.
    </p>
  </footer>
  d
  3
  .
  m
  i
  n
  .
  j
  s


</body>
</html>